app/bundles/CategoryBundle/EventListener/CategorySubscriber.php

```diff
@@ -25,6 +30,8 @@ public static function getSubscribedEvents(): array
             CategoryEvents::CATEGORY_ON_BUNDLE_LIST_BUILD => ['onCategoryBundleListBuild', 0],
             CategoryEvents::CATEGORY_POST_SAVE            => ['onCategoryPostSave', 0],
             CategoryEvents::CATEGORY_POST_DELETE          => ['onCategoryDelete', 0],
+            CategoryEvents::CATEGORY_PRE_DELETE           => ['onCategoryPreDelete', 0],
+            CategoryEvents::CATEGORY_TYPE_ENTITY          => ['onCategoryTypeEntity', 0],
```

Comment:
This should be good enough:
```suggestion
            CategoryTypeEntityEvent::class          => ['onCategoryTypeEntity', 0],
```